File: Docker_2/csharp/fibonacci.cs
using System;
using System.Diagnostics;

class Program {
    static void Main() {
        Stopwatch sw = Stopwatch.StartNew();
        ulong result = Fibonacci(10000); // Se usa la variable para evitar warnings
        sw.Stop();

        Console.WriteLine(sw.ElapsedMilliseconds); // Solo imprime el tiempo en milisegundos
    }

    static ulong Fibonacci(int n) {
        ulong a = 0, b = 1;
        for (int i = 0; i < n; i++) {
            ulong temp = a;
            a = b;
            b = temp + b;
        }
        return a;
    }
}

File: Docker_2/java/Fibonacci.java
package java;

public class Fibonacci {
    public static long fibonacci(int n) {
        long a = 0, b = 1;
        for (int i = 0; i < n; i++) {
            long temp = a;
            a = b;
            b = temp + b;
        }
        return a;
    }

    public static void main(String[] args) {
        long start = System.nanoTime();
        long result = fibonacci(10000);
        long end = System.nanoTime();

        System.out.println((end - start) / 1_000_000); // Convert to ms
    }
}
